internal/pkg/skuba/kubeadm/configmap_test.go

```diff
@@ -221,6 +221,7 @@ func TestGetAPIEndpointsFromConfigMap(t *testing.T) {
 				t.Errorf("error not expected on %s, but an error was reported (%v)", tt.name, err)
 				return
 			}
+			sort.Strings(gotAPIEndpoints)
```

Comment:
just curious, why sorting here?

Reply:
Explanation in the commit message here:

https://github.com/SUSE/skuba/pull/845/commits/c2450fc08f0be4a05bb923c58372f568dfdbaf6b